test/continuous.jl

```diff
@@ -58,9 +58,11 @@ for distr in [
     NoncentralF(2,2,2),
     NoncentralF(8,10,5),
     NoncentralT(2,2),
-    NoncentralT(10,2),
+    NoncentralT(10.,2),
```

Comment:
I can revert. I think this was a holdover from before I had code to test all the promoting constructors.